pkgs/development/python-modules/pythran/default.nix

```diff
@@ -62,14 +55,7 @@ in buildPythonPackage rec {
     "pythran.spec"
   ];
 
-  checkInputs = [
-    pytestCheckHook
-    numpy
-    scipy
```

Comment:
If we drop this, we should also drop them from the inputs if they are unused.

Reply:
I did.